tests/test_protocols.py

```diff
@@ -106,3 +113,83 @@ def test_invalid_message(self):
     #     sub = protocol.subscribe('python_test')
     #     for message in sub:
     #         assert message == self.valid
+
+
+class TestValidationErrorPublisher(TestCase):
+    @classmethod
+    def setUp(cls):
+        cls.schema = {
+            'required': ['foo']
```

Comment:
Shouldn't this be `'required': ['familyName']`?

Reply:
Yeah, this `cls.schema` thing doesn't work anymore. Just using that public example one now.

Reply:
Thanks though, will clean this up.